src/analytics/Properties.tsx

```diff
@@ -1341,6 +1341,13 @@ interface AssetsEventsProperties {
         description: string
         balanceUsd: number
       }
+    | {
+        assetType: 'token'
+        tokenId: string
+        title: string // Example: 'cUSD'
+        description: string
+        balanceUsd: number
```

Comment:
can we include `networkId` and `address` explicitly? It is part of the `tokenId` but the value is meant to be opaque.

Reply:
It looks like address is possibly undefined currently.

Reply:
still seems worth adding, for instance if we wanted to join on an existing table that uses only addresses